new_platforms/vsextension/ProjectTemplates/oeenclave/sub.mk

```diff
@@ -1,30 +1,35 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
-include $(NEW_PLATFORMS_PATH)oe_sub.mk
 
-global-incdirs-y += $(NEW_PLATFORMS_PATH)Inc
-global-incdirs-y += optee
+OE_SDK_PATH=../../packages/openenclave.0.2.0-CI-20190123-030613
+OE_SDK_INC_PATH=$(OE_SDK_PATH)/build/native/include
+OEEDGER8R=$(OE_SDK_PATH)/tools/oeedger8r.exe
```

Comment:
The new version of the NuGet package I'm working on will have the Linux version of `oeedger8r` which doesn't have an extension. That way we can ensure this Makefile can also be consumed from Linux and not only WSL.

Reply:
removed .exe